src/trading/simulation/simulate-sell.js

```diff
@@ -32,6 +34,11 @@ function simulateSell(outcome, sharesToCover, shareBalances, tokenBalance, userA
     }
   }
 
+  // Worst-case settlement fee for the seller is the max market price is hit and
+  // the seller must pay all the fees. This is equivilent to the taker short price going
+  // to 0, which we can calculate using calculateSettlementFee
+  simulatedSell.maxSettlementFees = calculateSettlementFee(sharesToCover, marketCreatorFeeRate, maxPrice.minus(minPrice), shouldCollectReportingFees, reportingFeeRate, 0);
```

Comment:
The goal is not to calculate the maximum possible settlement fee for this market, but rather, *if* the user is creating (making) a new order, to calculate what the maximum possible settlement fee they *could* end up paying *for that particular order*.